docs/REST.md

```diff
@@ -202,6 +203,37 @@ Content-Type: application/json
     }
 }
 ```
+
+## PATCH `/api/model/instock`
+
+Updates an unserialized model's total and instock values after adding a new one
+
+### Parameters
+
+* `modelAddress`: Address of the model to increment
```

Comment:
We've been doing this in several places. I'd rather just use `address`, but it's not :consus: unless we're a little inconsistent.